insights/parsers/tests/test_dmsetup_info.py

```diff
@@ -16,6 +18,11 @@
 VG00-opt           253   7 L--w    1    4      0 LVM-gy9uAwD7LuTIApplr2sogbOx5iS0FTaxIiCYm5hcvgQdXynPGBfHQLrtE3sqUKT2
 """.strip()
 
+DMSETUP_INFO_2 = """
+Name               Maj Min Stat Open Targ Event  UUID
+VG00-tmp           253   8 xIsr    1    1      a LVM-gy9uAwD7LuTIApplr2sogbOx5iS0FTax6lLmBji2ueSbX49gxcV76M29cmukQiw4
```

Comment:
It that output genuine?  How did you get 'a' to appear in the Event column?

Reply:
and how do you get 'x' in the first Stat column?

Or is this a test to check how it handles corrupt input? (If so, maybe add a comment)

Reply:
Its just fabricated data for testing @kergon